dev/io.openliberty.jakarta.validation.3.1_fat_tck/build.gradle

```diff
@@ -0,0 +1,14 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+ // Always pull from Maven Central so we don't have to upload transitive dependencies to artifactory
+apply from: '../wlp-gradle/subprojects/maven-central-mirror.gradle'
```

Comment:
We are no longer pulling in any outside runtime dependencies using Gradle (like Derby or TestNG)
This can be removed.
```suggestion

```

Reply:
@KyleAure Without this line the whole file can be deleted, right?

Reply:
deleted the file and retested. All tests are running successful